components/BottomNavigation/tests/unit/BottomNavigationBarColorThemerTests.m

```diff
@@ -31,6 +32,22 @@ @interface BottomNavigationBarColorThemerTests : XCTestCase
 
 @implementation BottomNavigationBarColorThemerTests
 
+- (void)testColorScheming {
+  // Given
+  MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];
+  colorScheme.primaryColor = [UIColor redColor];
+  colorScheme.onPrimaryColor = [UIColor blueColor];
+  MDCBottomNavigationBar *bottomNavigationBar = [[MDCBottomNavigationBar alloc] init];
+
```

Comment:
Set `bottomNavigationBar.barTintColor` and `bottomNavigationBar.selectedItemTintColor` to something specific here as well so that if ever — for some wacky reason — change the defaults to red and blue then these tests won't be no-ops.

Reply:
Done!